foo_053_alerts_price_spikes/contents/src/SamplePythonDataBridgesCall.py

```diff
@@ -64,7 +64,7 @@ def _refresh_token(self, scopes):
             raise TokenScopeError(f'Could not acquire requested scopes: {scopes}')
         self.tokens_by_scopes[scopes] = resp_data['access_token']
 
-    @backoff.on_exception(backoff.expo, (ApiServerError, ApiNotAuthorizedError), max_tries=5)
+    @backoff.on_exception(backoff.expo, (ApiServerError, ApiNotAuthorizedError), max_tries=8, base=10)

```

Comment:
@weiqi-tori how log does this make the backoff time?

Reply:
Maximum 8 tries, backoff 10 seconds on each try